pyfr/plugins/integrate.py

```diff
@@ -172,6 +165,29 @@ def _eval_exprs(self, intg):
 
         return intvals
 
+    def _grad_pvars(self, intg):
+        # Fetch the corrected gradients
+        grad_soln = intg.grad_soln
+
+        grads_eles = []
+        # Iterate over each element type in the simulation
+        for i, (soln, eleinfo) in enumerate(zip(intg.soln, self.eleinfo)):
+            plocs, wts, eset, emask = eleinfo
```

Comment:
eset, emask = eleinfo[2:] as I don't think we need the plocs of wts here.

Reply:
Fixed